packages/7-todo-tdd/src/components/todo-app/TodoApp.test.driver.tsx

```diff
@@ -0,0 +1,74 @@
+import { ReactWrapper } from 'enzyme';
+import TodoApp, { s as appStyles } from './TodoApp';
+import { s as todoItemStyles } from '../todo-item/TodoItem';
+import { wrapperGenerator } from '../../styles/utils';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import { ITodoMap } from 'fed-todo-journey_todo-common';
+import { s as commonStyles } from '../../styles/commonClasses';
+import { todosService } from '../../services/TodoService';
+import {TodosService } from '../../__mocks__/TodoService'
+
+const c = wrapperGenerator('.');
```

Comment:
what's this?

Reply:
i had to put "." before every jss class that i wanted to select with enzyme, i prefered to use a function instead of template strings

Reply:
overkill imo :)